Misc/NEWS

```diff
@@ -261,6 +261,7 @@ Library
 
 - bpo-29623: Allow use of path-like object as a single argument in
   ConfigParser.read().  Patch by David Ellis.
+- bpo-29649: struct.pack_into check boundary error message didn't respect offset.
```

Comment:
The entry should be added at the start of corresponding section. Separate it with empty lines from other text.

Add `()` after `struct.pack_into`. Add "Patch by <yourname>".